nix/devShells.nix

```
@@ -0,0 +1,33 @@
+{ inputs, ... }:
+{
+  default =
+    { lib, ... }@pkgs:
+    let
+      nnl = inputs.nnl.packages.${pkgs.system}.default;
+      
+      buildPackages = with pkgs; [
```

Equinox forces Nim to use clang. I think you should add that as a build dependency.

---

```suggestion
      stdenv = lib.mkForce pkgs.clangStdenv;
```

It's already in the file.

---

https://github.com/equinoxhq/equinox/blob/48a7339aedf4f4febee9b5d5bb2ff355da4cb81e/nix/devShells.nix#L26

It's already there.